src/qt/trafficgraphwidget.cpp

```diff
@@ -153,10 +155,10 @@ void TrafficGraphWidget::updateRates()
     update();
 }
 
-void TrafficGraphWidget::setGraphRangeMins(int mins)
+void TrafficGraphWidget::setGraphRange(std::chrono::minutes mins)
 {
-    nMins = mins;
-    int msecsPerSample = nMins * 60 * 1000 / DESIRED_SAMPLES;
+    m_range = std::chrono::minutes(mins);
+    auto msecsPerSample = m_range * 60 * 1000 / DESIRED_SAMPLES;
```

Comment:
https://github.com/bitcoin-core/gui/pull/524#pullrequestreview-853590647
> Tested [9bb48bf](https://github.com/bitcoin-core/gui/commit/9bb48bf657c7f2d36309807fbc7b55d0ab581a76) on Linux Mint 20.3 (Qt 5.12.8) -- the "Network Traffic" tab shows a black window.

Could be fixed with
```suggestion
    const auto msecs_per_sample{std::chrono::duration_cast<std::chrono::milliseconds>(m_range) / DESIRED_SAMPLES};
```

Reply:
Thanks, @hebasto for catching it and giving suggestions on how to correct this misbehavior

Addressed in 5e6ad435b0ed634fca5745668bf949799b306bff